aten/src/ATen/native/Resize.cpp

```diff
@@ -0,0 +1,43 @@
+#include "ATen/ATen.h"
+#include "TH/THTensor.hpp"
+
+namespace at { namespace native {
+
+Tensor& resize__cpu(Tensor& self, IntList size) {
```

Comment:
The return of the dastardly double underscore. (I laughed when I found out our codegen handles both naming conventions.)

Reply:
As in, our codegen handles `resize__cpu` and `resize_cpu_`?

Reply:
yep